### ISSUE TITLE
Stack of objects

### ISSUE THREAD

**Issue by inkfood** [schteppe/p2.js#154]
How to create a stable stack of objects, without having a world.solver.iterations rate of 100.
Here an example:
http://jsfiddle.net/inkfood/kevcwu1w/

As you can see all the boxes get compressed in first few frames, after that they start shifting around.
While the shifting problem can easily be solved with the bodySleep option, how can I prevent the compression in the first place? Even with a higher world.solver.iterations the compression is still visible.
How to avoid that behavior?     

![p2stack](https://cloud.githubusercontent.com/assets/8083536/7275966/16ad2afe-e907-11e4-91e8-6319df19f5ca.gif)

**Comment by schteppe** [comment]
Tough problem to solve. Try:

* Less gravity
* dropping the boxes one by one and not all at once
* cheat a bit: drop them one by one and make boxes at rest static
* contact stiffness and relaxation parameters
* merge some body pairs into one body - reduces contacts

What are you building, a stacking game?

**Comment by inkfood** [comment]
I already played with all the values you mentioned above. I just thought I might miss something crucial, like an init phase or a pre update step. I see, thats just how the physics engine works. Anyway I got the best results with this values
```
world.setGlobalStiffness(1e6);
world.solver.iterations = 30;
world.defaultContactMaterial.relaxation= 1;
world.defaultContactMaterial.restitution= 0.001;
```

I just do some prototyping, it might end up in a stacking game, who knows : ).
Take a look at: http://www.inkfood.com/2d-physics-in-3d-space/

Solved, thank you, keep up the good work!!
